app/views/merchants/index.html.erb

```diff
@@ -4,11 +4,31 @@
   <% @merchants.each do |merchant| %>
     <section id='merchant-<%= merchant.id %>'>
       <ul>
+        <% if admin_user? %>
+        <li><%= link_to merchant.name, admin_merchant_path(merchant) %></li>
+        <% else %>
         <li><%= link_to merchant.name, merchant_path(merchant) %></li>
+        <% end %>
         <li><%= merchant.city%></li>
         <li><%= merchant.state%></li>
         <li><%= merchant.created_at%></li>
+        <% if admin_user? %>
+        <li><%= link_to 'Disable' %></li>
+        <% end %>
```

Comment:
Maybe we could move the link to disable, into the if admin_user? block on line 7?

Reply:
Second.  Unless it's a styling reason, the less if blocks the better imo.

Reply:
Did make the choice for styling, for sure doesn't matter. Ill move it